packages/common/src/ui/components/buttons/ButtonShowcase.stories.tsx

```diff
@@ -72,17 +75,7 @@ const Template: Story = () => {
         <Wrapper text="Base Button: Contained variant, primary color">
           <BaseButton
             variant="contained"
-            color="primary"
-            onClick={getOnClick('Base button')}
-          >
-            Base Button
-          </BaseButton>
-        </Wrapper>
-
-        <Wrapper text="Base Button: Contained variant, secondary color">
-          <BaseButton
-            variant="contained"
-            color="secondary"
+            color={color}
```

Comment:
Should probably change the description for the buttons, to remove `primary` now that they aren't hard coded to the primary colour